topics/proteomics/tutorials/neoantigen-2-non-normal-database-generation/tutorial.md

```diff
@@ -0,0 +1,666 @@
+
+> <hands-on-title> INDEL - FASTA-to-Tabular </hands-on-title>
+>
+> 1. {% tool [FASTA-to-Tabular](toolshed.g2.bx.psu.edu/repos/devteam/fasta_to_tabular/fasta2tab/1.1.1) %} with the following parameters:
+>    - {% icon param-file %} *"Convert these sequences"*: `output_indel` (output of **CustomProDB** {% icon tool %})
+>
+>
+{: .hands_on}
+
+> <hands-on-title> SNV - FASTA-to-Tabular </hands-on-title>
+>
+> 1. {% tool [FASTA-to-Tabular](toolshed.g2.bx.psu.edu/repos/devteam/fasta_to_tabular/fasta2tab/1.1.1) %} with the following parameters:
+>    - {% icon param-file %} *"Convert these sequences"*: `output_snv` (output of **CustomProDB** {% icon tool %})
+>
+>
+{: .hands_on}
+
+> <hands-on-title> RPKM - FASTA-to-Tabular </hands-on-title>
+>
+> 1. {% tool [FASTA-to-Tabular](toolshed.g2.bx.psu.edu/repos/devteam/fasta_to_tabular/fasta2tab/1.1.1) %} with the following parameters:
+>    - {% icon param-file %} *"Convert these sequences"*: `output_rpkm` (output of **CustomProDB** {% icon tool %})
+>
+>
+{: .hands_on}
+
+> <question-title></question-title>
+>
+> 1. Why is it helpful to convert FASTA sequences to a tabular format?
+> 2. What information is typically retained when converting a FASTA file to tabular format?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. Converting FASTA sequences to tabular format makes it easier to visualize, filter, and manage the data. The tabular structure allows researchers to perform sorting, searching, and data manipulation more efficiently, which is beneficial for downstream analysis.
+> > 2. The conversion usually retains the sequence identifiers and the sequences themselves. Each sequence entry in the FASTA file is represented in two columns: one for the identifier and another for the actual sequence, enabling easy access to both key elements of the data.
+> >
+> {: .solution}
+>
+{: .question}
+
+## Manipulating the headers with Column Regex Find And Replace
+
+Column Regex Find And Replace is a tool that applies regular expression (regex) patterns to specified columns in a tabular dataset to find and replace text patterns. In this task, the tool is used on the output from the FASTA-to-Tabular step to standardize or format specific patterns in the data. By applying regex patterns to column c1, the tool identifies and modifies specific sequence identifiers or annotations, making them easier to interpret or use in further analyses.
+
+In this workflow, Column Regex Find And Replace cleans and formats the data in a way that makes identifiers or variant descriptions consistent. This is important for data compatibility, especially when the data needs to be used across different tools or integrated into larger datasets. It ensures that all sequence labels or variant annotations follow a uniform format, which reduces errors in downstream analyses.
+
+
+> <hands-on-title> INDEL-Column Regex Find And Replace </hands-on-title>
```

Comment:
for these 3 hands-on boxes, I would include a step at the end to rename the output files, to make it a bit easier in the tabular-to-fasta step to indicate which files should be used as input